explorer/app/components/Page/Page.styles.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from "@emotion/styled";
+
+export const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+export const Content = styled.div`
+  display: flex;
+  flex-direction: column;
+  position: relative;
```

Comment:
We can now remove the relative position on the content div with the changes to how the Header mobile modal is rendered 😄